theme_vehicle/views/customizations.xml

```diff
@@ -1,171 +1,365 @@
+            <attribute name="class" remove="btn-lg" separator=" "/>
+        </xpath>
+    </template>
+
+    <!-- ======== PICTURE ======== -->
+    <template id="s_picture" inherit_id="website.s_picture" name="Vehicle s_picture">
+    <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pt0 pb0 oe_img_bg" remove="pt48 pb24" separator=" "/>
+        </xpath>
+        <xpath expr="//section" position="attributes">
+            <attribute name="style" add="background-image: url(/web/image/website.s_picture_default_image); background-position: 50% 0;"/>
+        </xpath>
+        <!-- Layout -->
+        <xpath expr="//div[hasclass('container')]" position="replace">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-lg-12 oe_img_bg o_bg_img_center text-center d-flex flex-column" style="height:850px;">
+                        <div class="o_we_bg_filter" style="background-color: rgba(40,40,40,0.50)"></div>
+                        <div class="pt40 d-flex flex-column flex-grow-1">
+                            <h2 class="text-o-color-3">View the all-new KORAN in 3D wherever you are<br/></h2>
+                            <p class="text-o-color-3">Bring the KORAN to life on your smartphone or tablet so you can visualise it for yourself.<br/></p>
+                        </div>
+                        <div class="pb40 d-flex justify-content-center align-items-end">
+                            <span class="text-o-color-3">Our uniquely designed LED headlights are not only gorgeous but powerfully light your way.<br/></span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </xpath>
+    </template>
+
+    <!-- ======== THREE COLUMNS ======== -->
+    <template id="s_three_columns" inherit_id="website.s_three_columns">
+        <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="o_cc3 pt48 pb48" remove="o_cc2 pt32 pb32 bg-200" separator=" "/>
+        </xpath>
+        <!-- Card -->
+        <xpath expr="(//div[hasclass('card')])[1]" position="attributes">
+            <attribute name="class" add="border-0" separator=" "/>
+            <attribute name="style">box-shadow: rgba(0, 0, 0, 0.20) 0px 4px 16px 0px !important;</attribute>
+        </xpath>
+        
+        <xpath expr="(//div[hasclass('card')])[2]" position="attributes">
+            <attribute name="class" add="border-0" separator=" "/>
+            <attribute name="style">box-shadow: rgba(0, 0, 0, 0.20) 0px 4px 16px 0px !important;</attribute>
+        </xpath>
+        <xpath expr="(//div[hasclass('card')])[3]" position="attributes">
+            <attribute name="class" add="border-0" separator=" "/>
+            <attribute name="style">box-shadow: rgba(0, 0, 0, 0.20) 0px 4px 16px 0px !important;</attribute>
+        </xpath>
```

Comment:
This is buggy you force border-0 -> making it apparently impossible to control the border with the editor (because border-0 remove the border style too). Maybe you could remove the border by setting an inline border-width...